spec/exercises.js

```diff
@@ -3,6 +3,12 @@ var assert = require('assert')
 var path = require('path')
 var _ = require('lodash')
 var wUtil = require('workshopper-adventure/util')
+var config
+try {
+  config = require(`${process.cwd()}/.workshopper-adventure-test-config.js`)
```

Comment:
A) this uses template strings, see my comment about destructuring
B) As the config has defaults it would be good to define all defaults here.

```const config = Object.assign({}, requireSafe(...)```

Reply:
It's a good idea to define defaults here.

Also not sure config file name is the best. Maybe should rename to something like `workshopper-test.js`?

Reply:
Hmm.. `workshopper-test.js` is a bit less easy to comprehend than `workshopper-adventure-test.config.js`. I was okay with the long name as it is straight forward. A middleground would be `workshopper-test.config.js`